test.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+set -e
+
+function printUsage() {
+  echo "Usage: test.sh <repository> <username>"
+  echo "Example: test.sh compucorp/rse davialexandre"
+  echo "The script expects the following environment variables to be defined in order for it to run:"
+  echo "  GITHUB_TOKEN: a personal access token from the given user with permissions to clone and create pull requests to the given repository"
+  echo "  AWS_ACCESS_KEY: a access key with permission to read from the anonymized-db S3 bucket used by the mysql-anondb docker image"
+  echo "  AWS_SECRET_KEY: the secret key for the given AWS_ACCESS_KEY"
+}
+
+function getContainerStatus() {
+  docker inspect --format="{{if .Config.Healthcheck}}{{print .State.Health.Status}}{{end}}" "$1"
+}
+
+[ -z "$1" ] && echo "Missing repository!" && printUsage && exit 1
+[ -z "$2" ] && echo "Missing username!" && printUsage && exit 1
+[ -z "$GITHUB_TOKEN" ] && echo "Missing GITHUB_TOKEN" && printUsage && exit 1
+[ -z "$AWS_ACCESS_KEY" ] && echo "Missing AWS_ACCESS_KEY" && printUsage && exit 1
+[ -z "$AWS_SECRET_KEY" ] && echo "Missing AWS_ACCESS_KEY" && printUsage && exit 1
+
+set -x
+
+repository=$1
+github_username=$2
+
+## Clone the repository following an approach similar to what https://github.com/actions/checkout does
+workdir="$PWD/workdir/$repository"
+mkdir -p "$workdir"
+cd "$workdir"
+git init
+git remote add origin "https://$github_username:$GITHUB_TOKEN@github.com/$repository"
+git -c protocol.version=2 fetch --prune --progress --no-recurse-submodules origin "+refs/heads/*:refs/remotes/origin/*" "+refs/tags/*:refs/tags/*"
+git checkout --progress --force -B master refs/remotes/origin/master
+
+# We need the FROM_SITE in order to start the mysql-anondb container
+from_site=$(grep -Eoh "FROM_SITE: (.+?)" "$workdir"/.github/workflows/* | cut -d ' ' -f 2)
+[ -z "$from_site" ] \
+  && echo "Couldn't extract the FROM SITE from the workflow file. Please check if this project has been configured for automated security updates" \
+  && exit 1
+
+# Some projects will need the PHP 5.6 or the action and some other will use 7.2
```

Comment:
I think I understand what you mean, but the sentence structure in the first line looks wrong to me.